AncientReligions/events/ACR_other_event.txt

```diff
@@ -213,4 +212,207 @@ character_event = {
 		ai_chance = { factor = 1 }
 		clr_character_flag = gone_pagan
 	}
-}
\ No newline at end of file
+}
+
+# Ask the player if they want to "paganify" the world
+character_event = {
+	id = ancrel.9001
+	desc = ancrel.9001.desc
+	is_triggered_only = yes
+
+	trigger = {
+		NOT = { has_global_flag = acr_paganify }
+	}
+
+	immediate = {
+		set_global_flag = acr_paganify
+	}
+
+	option = {
+		name = ancrel.9001.a
+		any_independent_ruler = {
+			any_realm_character = {
```

Comment:
Could there be a check so that already pagan characters keep their current religion ?
I'm thinking of zun_pagan which is not mapped and would become Zoroastrian.
Or if using another mod with some new pagan religions.